eras/conway/impl/cddl-files/conway.cddl

```diff
@@ -147,13 +147,13 @@ vote = 0 .. 2
 
 gov_action_id =
   [ transaction_id   : $hash32
-  , gov_action_index : uint
+  , gov_action_index : uint .size 4
   ]
 
 required_signers = nonempty_set<addr_keyhash>
 
 transaction_input = [ transaction_id : $hash32
-                    , index : uint
+                    , index : uint .size 8
```

Comment:
In the matter of fact we need to change the decoder for this to be `Word16` starting with protocol version 9. Could you do this in this PR as well? Make sure to change only the decoder, not the type itself.

If you can do the same for `CertIx` too that would be awesome.

```suggestion
                    , index : uint .size 2
```